lib/generators/react_on_rails/templates/base/base/config/webpack/test.js.tt

```diff
@@ -0,0 +1,6 @@
+<%= add_documentation_reference(config[:message], "// https://github.com/shakacode/react_on_rails_tutorial_with_ssr_and_hmr_fast_refresh/blob/master/config/webpack/development.js") %>
```

Comment:
@justin808 Minor one, should the doc reference link to https://github.com/shakacode/react_on_rails_tutorial_with_ssr_and_hmr_fast_refresh/blob/master/config/webpack/test.js instead or is development.js fine because we're essentially loading it as is? Might need to see if any relevant updates are needed in the tutorial repo

Reply:
@tomdracz I'm wondering if we can refactor the sort of convoluted current setup a bit. Think about what would be the simplest JS code to get the job done, maybe depending on a small NPM helper package, like react-on-webpacker.

Reply:
@tomdracz we actually need some tests in that sample repo, so then we'd have a file called test.js.

Having a file named `webpack/<ENV>.js` is the old webpacker style, ditched in favor of a simpler webpack.config.js.

Reply:
I updated! for the 13.0.1 release!